Assets/Scripts/Battle/Begin.cs

```diff
@@ -10,10 +10,15 @@ public Begin(BattleSystem battleSystem) : base(battleSystem)
 
         public override IEnumerator Start()
         {
-            battleSystem.Player.transform.SetParent(battleSystem.Board.GetTileTransform(battleSystem.Player.coordinates));
+            battleSystem.Player.transform.SetParent(
+                battleSystem.Board.GetTileTransform(battleSystem.Player.coordinates));
             battleSystem.Player.transform.position =
                 battleSystem.Board.GetPositionForTileEntityFromCoordinate(battleSystem.Player.coordinates);
-            
+
+            battleSystem.PlayerEnergy.SetText(
+                $"Energy: {battleSystem.Player.energy = 5}"
```

Comment:
= 5? co to tu robi?

Reply:
początkowa wartość energii gracza? W sumie mogę to pominąc i na sztywno w klasie fighter to ustawic